src/main/java/org/microg/nlp/backend/nominatim/BackendService.java

```diff
@@ -42,11 +44,18 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
-import java.util.Objects;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import static android.os.Build.VERSION.RELEASE;
+import android.os.Parcel;
+import android.os.PowerManager;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
```

Comment:
Please ensure that imports are formatted properly (i.e. ordered by name, one line between different first level package name)

Reply:
fixed